src/components/structures/RoomView.js

```diff
@@ -516,21 +518,26 @@ module.exports = React.createClass({
         }
     },
 
-    _updateTabCompleteList: new rate_limited_func(function() {
+    _updateTabCompleteList: function() {
         var cli = MatrixClientPeg.get();
+        console.log('_updateTabCompleteList');
+        console.log(this.state.room);
+        console.trace();
 
-        if (!this.state.room || !this.tabComplete) {
+        if (!this.state.room) {
             return;
         }
         var members = this.state.room.getJoinedMembers().filter(function(member) {
             if (member.userId !== cli.credentials.userId) return true;
         });
+
+        UserProvider.getInstance().setUserList(members);
```

Comment:
This doesn't seem to work after loading the page until you switch tabs, although it seems this is not a new bug. It would be good to fix if possible.